bool visible
bool vertical # true implies vertical, false implies horizontal
bool open # true implies open, false implies closed
geometry_msgs/Vector3 position
